pinliner/importer.template

```diff
@@ -87,7 +87,7 @@ class InlinerImporter(object):
 
         try:
             compiled_code = self.get_code(fullname, module.__file__)
-            exec compiled_code in module.__dict__
+            exec(compiled_code, module.__dict__)
```

Comment:
This breaks Python 2 compatibility. We need conditional execution based on the Python version that this is running.
We can do an if ... else clause on each exec call:
```python
class InlinerImporter(object):
    [ ... ]
    def load_module(self, fullname):
        [ ... ]
        if sys.version_info.major == 2:
            exec compiled_code in module.__dict__
        else:
            exec(compiled_code, module.__dict__)
```

Or we can create a custom method for Python 2 and a variable to hold the method we want to use based on the Python version:

```python
class InlinerImporter(object):
    [ ... ]
    @staticmethod
    def py2_exec(code, dictionary):
        exec code in dictionary
    my_exec = py2_exec if sys.version_info.major == 2 else exec
    
    def load_module(self, fullname):
        [ ... ]
        self.my_exec(compiled_code, module.__dict__)
```

Reply:
Python 2 supports both forms, so this should work just fine as-is.

Reply:
You are totally right.

I thought it did support both forms, but *I didn't want to think*, so I run a little test for `exec` when reviewing your code, but did something wrong, because I got an error, which now I don't.